code/datums/diseases/advance/symptoms/rodman.dm

```diff
@@ -0,0 +1,124 @@
+						"Your thoughts seem to be clearing up.",
+						"Your brain feels great.",
+						"The fog in your mind is clearing up.")]</span>")
+					return
+				if((increase_volume == TRUE) && (prob(35)))
+					M.say( pick( list(
+					"Ook!!", 
+					"Ook.", 
+					"Ooh-ooh!!",
+					"Ah-ah!!",
+					"Eek-eek!!",
+					"Hoo hoo!!", 
+					"SCREE!!") )) //More volume means shouted messages now
+					playsound(M.loc, pick( list('sound/creatures/monkey/monkey_screech_1.ogg','sound/creatures/monkey/monkey_screech_2.ogg','sound/creatures/monkey/monkey_screech_3.ogg','sound/creatures/monkey/monkey_screech_4.ogg','sound/creatures/monkey/monkey_screech_5.ogg','sound/creatures/monkey/monkey_screech_6.ogg','sound/creatures/monkey/monkey_screech_7.ogg')), 100, 1)
+					return
+				if(prob(20))
+					M.say( pick( list(
+					"Ook!",
+					"Ook.",
+					"Eek", 
+					"Oop?", 
+					"Aak-eek.", 
+					"Chee."
+					) )) //Return to monke.
+					playsound(M.loc, pick( list('sound/creatures/monkey/monkey_screech_1.ogg','sound/creatures/monkey/monkey_screech_2.ogg','sound/creatures/monkey/monkey_screech_3.ogg','sound/creatures/monkey/monkey_screech_4.ogg','sound/creatures/monkey/monkey_screech_5.ogg','sound/creatures/monkey/monkey_screech_6.ogg','sound/creatures/monkey/monkey_screech_7.ogg')), 50, 1)
+			if((!ishuman(M)) && (awaken_monkey == TRUE) && (is_monkey == TRUE)) //Confirm they were a monkey from the start, currently a monkey and that the threshold is hit.
+				awaken_monkey = FALSE // Only one attempt at this, to minimize spam.
+				if((!M.mind) && (isliving(M))) //Confirm they are mindless and alive
+					//Give them a name that isn't just Monkey(420)
+					M.name = pick(world.file2list("strings/random_monkey_names.txt"))
+					if(M.name == "Tim the Sorcerous" || "Ook Operative" || "Sherlok Monke" || "Monkie Holms" || "Monkey-anda Jones" || "Ook") 
+					//Checking for ANY of the unique modifier names in one go, rather than a set of many 'if' statements				
+						if(M.name == "Tim the Sorcerous") //Wizard Hat
+							var/obj/item/clothing/C
+							C = new /obj/item/clothing/head/wizard(M)
+							ADD_TRAIT(C, TRAIT_NODROP, DISEASE_TRAIT)
+							M.equip_to_slot_or_del(C, ITEM_SLOT_HEAD)
+						if(M.name == "Ook Operative") //Fake nuke op space helmet
+							var/obj/item/clothing/C
+							C = new /obj/item/clothing/head/syndicatefake(M)
+							ADD_TRAIT(C, TRAIT_NODROP, DISEASE_TRAIT)
+							M.equip_to_slot_or_del(C, ITEM_SLOT_HEAD)
+						if(M.name == ("Sherlok Monke" || "Monkie Holms" || "Monkey-anda Jones")) //Detective fedora
+							var/obj/item/clothing/C
+							C = new /obj/item/clothing/head/fedora/det_hat(M)
+							ADD_TRAIT(C, TRAIT_NODROP, DISEASE_TRAIT)
+							M.equip_to_slot_or_del(C, ITEM_SLOT_HEAD)
+						if(M.name == "Ook") //Unique hat for Ook
+							var/obj/item/clothing/C
+							C = new /obj/item/clothing/mask/ookmask(M)
+							ADD_TRAIT(C, TRAIT_NODROP, DISEASE_TRAIT)
+							M.equip_to_slot_or_del(C, ITEM_SLOT_MASK)

```

Comment:
You should probably just use a switch statement here

Reply:
Switch added for the name, that does clean it up a lot